examples/fluids/qfunctions/shocktube.h

```diff
@@ -300,14 +301,14 @@ CEED_QFUNCTION(EulerShockTube)(void *ctx, CeedInt Q,
       // Approximate dot(j_vec,grad(N)) using the metric tensor
       for (int j=0; j<3; j++)
         j_gradn[j] = j_vec[0] * dXdx[0][j]
-                   + j_vec[1] * dXdx[1][j]
-                   + j_vec[2] * dXdx[2][j];
+                     + j_vec[1] * dXdx[1][j]
+                     + j_vec[2] * dXdx[2][j];
```

Comment:
This is more consistent with other terms and I think it's easier to read.
```
      for (int j=0; j<3; j++)
        j_gradn[j] = j_vec[0] * dXdx[0][j] +
                     j_vec[1] * dXdx[1][j] +
                     j_vec[2] * dXdx[2][j];
```

Reply:
`make style` consistently yells at me about this. I don't know what I should be doing to make it accept the formatting. Do you have any ideas?

Reply:
You can add style exclusion tags.

LCOV_EXCL_START 
And
LCOV_EXCL_STOP

Reply:
You can also put the + on the preceding line.

Reply:
I don't recall astyle being able to handle the `+` at the beginning of the line, but I think it's easier to read that way. I'd just ignore style here, or write it as a loop. Maybe we can use clang-format some day.

Reply:
I'll move the + sign, I missed that detail in Leila's original suggestion